roles/pulp3-redis/vars/CentOS.yml

```diff
@@ -2,4 +2,3 @@
 pulp_preq_packages:
   - python36
   - python36-devel
-pulp_python_interpreter: /usr/bin/python36
```

Comment:
Already defined in the "pulp" role, so I don't think it's necessary to copy here?